From	Krzysztof Mazur <>
Subject	[PATCH] ssb: Reenable PCI host on !MIPS
Date	Sat, 27 Jan 2018 00:58:49 +0100

The commit 58eae1416b804d900014d84feadda7195007cc30
("ssb: Disable PCI host for PCI_DRIVERS_GENERIC") disabled
CONFIG_SSB_PCIHOST and CONFIG_SSB_B43_PCI_BRIDGE, which are
required for SSB-based b43 PCI cards, on everything except MIPS
with PCI_DRIVERS_LEGACY, including x86 (only MIPS defines
that config option). Reenable it.

Signed-off-by: Krzysztof Mazur <krzysiek@podlesie.net>
---
 drivers/ssb/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ssb/Kconfig b/drivers/ssb/Kconfig
index 71c73766ee22..3dc57414e06b 100644
--- a/drivers/ssb/Kconfig
+++ b/drivers/ssb/Kconfig
@@ -32,7 +32,7 @@ config SSB_BLOCKIO
 
 config SSB_PCIHOST_POSSIBLE
 	bool
-	depends on SSB && (PCI = y || PCI = SSB) && PCI_DRIVERS_LEGACY
+	depends on SSB && (PCI = y || PCI = SSB) && (!MIPS || PCI_DRIVERS_LEGACY)
 	default y
 
 config SSB_PCIHOST
-- 
2.16.0.rc1.238.g530d649a7
